README.md

```diff
@@ -1,13 +1,9 @@
 # Project Happy Thoughts API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I have created my own API for posting new Happy Thoughts to the previous created React project Happy Thoughts. Built with MongoDB and Express.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+Old Happy Thoughts project with the new API: https://happy-thoughts-joanna.netlify.app/
+
+API link: https://joanna-project-happy-thoughts.herokuapp.com/thoughts
```

Comment:
```suggestion
API link: https://joanna-project-happy-thoughts.herokuapp.com
```
Maybe it would be better without the `/thoughts` at the end, so the user can see the link to live frontend (because now it sends us to the list of thoughts)